.github/workflows/ci.yml

```diff
@@ -44,16 +44,10 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Setup conda
-      uses: s-weigand/setup-conda@v1
-      with:
-        update-conda: true
-        python-version: ${{ matrix.python-version }}
-        conda-channels: anaconda, conda-forge
-    - name: Install ecCodes and MetView
+    - name: Install ecCodes for Grib support
       run: |
-        conda install -y eccodes>=2.21.0 -c conda-forge
-        conda install -y metview-batch -c conda-forge
+        sudo apt-get update
+        sudo apt-get --assume-yes install libeccodes-dev
```

Comment:
Is not include metview necessary for this change?

Reply:
Performed `git rebase` and resolved conflict errors.